aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,266 @@
+| Short written format           | {Intl.DateTimeFormat('en-US', { dateStyle: 'medium' }).format(date)}                                              | When space is limited.                                                                                                                                                                                                                                                                 | <Box>`Intl.DateTimeFormat('en-US', { dateStyle: 'medium' }).format(date)`</Box>                                              |
+| Numeric locale-specific format | {Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)}                | When the user needs to compare across dates, such as in a column or list. Otherwise, use written format. Numeric format can differ depending on the region, so even if the date is translated by the UI based on the user's locale, numeric dates can still cause confusion for users. | <Box>`Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)`</Box>                |
+| ISO 8601 format                | {date.toISOString()}                                                                                              | Log files; trigger times.                                                                                                                                                                                                                                                              | <Box>`date.toISOString()`</Box>                                                                                              |
+
+Other than log files, date and time should be presented according to the [user's locale](#locale-formatting) and [timezone](#timezones).
+
+For log files, there are two options:
+
+1. Always output timestamps in UTC, indicating UTC in the output.
+1. Allow the user to specify a default timezone and always output timestamps in that timezone, specifying the timezone in the output.
+
+### Locale formatting
+
+Date and time are formatted differently depending on the language and region. Some aspects that are locale-dependent include date and time ordering, separators, and 12-hour vs 24-hour clocks.
+
+There are various [Javascript Date methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) that can be leveraged to convert from UTC to the user's locale.
+
+Whenever possible, present date and time according to the user's locale.
+
+export const localeDate = new Date('2023-09-15T12:00:00.000Z');
+
+export const localeFormat = {
+  year: 'numeric',
+  month: 'long',
+  day: 'numeric',
+  weekday: 'long',
+};
+
+| Locale | Date                                                            |
```

Comment:
Would it be helpful to add a DateTime column as well?

Reply:
Assuming this means the code snippet? Yes I can add.

Reply:
Done.

Reply:
I was actually referring to presenting a localized date + time. The content above speaks to localizing date and time, so was thinking that in addition to the localized date column, we'd do a localized date and time column which would likely present concepts like 24 hour clock and timezone.

Reply:
Thanks for clarifying, will update so time is also include